backend/services/UserService.js

```diff
@@ -1,39 +1,49 @@
-const BaseModelService = require('./baseModelService');
+const BaseModelService = require('./BaseModelService');
 
 class UserService extends BaseModelService {
     constructor(bcryptjs) {
         super();
         this.bcryptjs = bcryptjs;
     }
 
-    static get table() {
+    get table() {
         return 'users';
     }
 
-    static register(email, password) {
+    register(email, password) {
         const salt = this.bcryptjs.genSaltSync(10);
         const encryptedPassword = this.bcryptjs.hashSync(password, salt);
 
         return this.create({
             email,
             password: encryptedPassword,
-            created_at: Date.now()
+            createdAt: Date.now()
         });
     }
 
-    static async login(email, password) {
+    async login(email, password) {
```

Comment:
as above => src/services/auth/LoginHandler.js
something like this

src/
node_modules/
tests/

this is also a standard in modern applications

Reply:
but of course do it as separate PR - like moving all files to new dir standards and don't wait for CR - just merge such PR